pinot-query-runtime/src/test/resources/queries/WindowFunctions.json

```diff
@@ -51,6 +51,50 @@
           [768]
         ]
       },
+      {
+        "description": "Single empty OVER() count(*) without select col",
+        "sql": "SELECT COUNT(*) OVER() FROM {tbl}",
```

Comment:
I am guessing this is in alignment with Postgres supports because MySQL does not support this kind of query -- `count(*) with empty over()` ?

Reply:
yes this query works in PostgreSQL:

```
dvdrental=# select ROW_NUMBER() OVER() from payment;
 row_number 
------------
          1
          2
          3
          4
          5
          6
          7
          8
          9
         10
         11
         12
         13
         14
         15
         16
         17
         18
         19
         20
         21
         
dvdrental=# select COUNT(*) OVER() from payment;
 count 
-------
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
 14596
```

Reply:
Got it. Thanks for clarifying.